### PR TITLE
feat: counter and home page

### PR DESCRIPTION
**Pull request by martlet-anastasia**

### CODE REVIEW

File: src/pages/Counter/containers/CounterContainer.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  handleIncrement = () => {
+    this.setState((state) => {
+      return {
+        countValue: state.countValue + 1,
+      };
+    });
+    this.setState((state) => {
+      return {
+        parityType: this.calculateParityType(state),
+      };
```
Comment (Boobaley) [inline src/pages/Counter/containers/CounterContainer.js]: you don't need this `setState`. you can do the same in the `setState` above

File: src/pages/Counter/containers/CounterContainer.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  handleDecrement = () => {
+    this.setState((state) => {
+      return {
+        countValue: state.countValue > 1 ? state.countValue - 1 : 0,
+      };
+    });
+    this.setState((state) => {
+      return {
+        parityType: this.calculateParityType(state),
+      };
```
Comment (Boobaley) [inline src/pages/Counter/containers/CounterContainer.js]: same as above. Also, check the `componentDidUpdate` lifecycle. Just imagine if you will have 10 handlers with keeping parity type logic. You will have to put this every time, but with `componentDidUpdate` you are able to write this once

File: src/scss/_colors.scss [hunk 3]
```diff
@@ -0,0 +1,6 @@
+$blue: #32a1ce;
+$orange: #FDB833;
+$white: #ffffff;
+$greyblack: #2C1A1D;
```
Comment (Boobaley) [inline src/scss/_colors.scss:4]: praise: Great!

**Review by Boobaley — CHANGES_REQUESTED** [review]

File: src/pages/Counter/containers/CounterContainer.js (hunk 2 above)
Comment (martlet-anastasia) [reply src/pages/Counter/containers/CounterContainer.js]: Move parity type to componentDidUpdate()

File: src/pages/Counter/containers/CounterContainer.js (hunk 1 above)
Comment (martlet-anastasia) [reply src/pages/Counter/containers/CounterContainer.js]: Move parity type to componentDidUpdate()

**Pull request merged**